config/secrets.exs.example

```diff
@@ -22,3 +22,16 @@ config :cadet,
 
 config :sentry,
   dsn: "https://public_key/sentry.io/somethingsomething"
+
+# Configure Arc File Upload for development only
```

Comment:
we have different config files for different environments eg dev and prod. could you move them to the appropriate files.

Reply:
I added it here because we didn't have a `dev.secrets.exs.example`. Refactored.